policyengine_canada/tests/gov/cra/tax/income/credits/gst_credit/gst_credit_base.yaml

```diff
@@ -0,0 +1,38 @@
+- name: Person amounts for family of 3
```

Comment:
We don't need a unit test for variables without formulas, like this which just uses `adds`, so you can delete this.